content/6_Advanced/XOR_Basis.mdx

```diff
@@ -25,8 +24,228 @@ frequency: 1
 	>
 		used at USACO Camp
 	</Resource>
+	<Resource
+		source="hackmd"
+		url="https://hackmd.io/@minhnguyent546/BJKcH_2ti#XOR-BASIS"
+		title="Minh Thien Nguyen - XOR BASIS"
+		starred
+	>
+		detailed explanation about XOR basis note: translate the page to english
+	</Resource>
```

Comment:
this resource is hard to understand even when translating to english... is there a specific point from the blog you want to add? if so, i think it's best that you write it in the module below.

Reply:
Done.